solutions/kubernetes/cortx-deploy.sh

```diff
@@ -104,6 +104,9 @@ function setup_cluster {
         echo "---------------------------------------[ Print Cluster Status ]----------------------------------------------"
         rm -rf /var/tmp/cortx-cluster-status.txt
         ssh -o 'StrictHostKeyChecking=no' "$master_node" '/var/tmp/cortx-deploy-functions.sh --status' | tee /var/tmp/cortx-cluster-status.txt
+
+        echo "---------------------------------------[ Collect CORTX Support Bundle Logs ]----------------------------------------------"
+        ssh -o 'StrictHostKeyChecking=no' "$master_node" '/root/deploy-scripts/k8_cortx_cloud/logs-cortx-cloud.sh -s /root/deploy-scripts/k8_cortx_cloud/solution.yaml'
```

Comment:
did you check if this stage runs even if cluster status fails?

Reply:
Changed with post action so that it will execute logs collection on artifacts only on build failure.